web/templates/page/index.html.eex

```diff
@@ -10,7 +10,9 @@
   <table class="w-80 center dn db-ns">
     <thead>
       <tr class="bb bw2 dwyl-b--gray">
-        <th class="pv3 pl2">Repo</th>
+        <th><%= sort_link @conn, @rummage, [field: :repo_name, name: "Repo"] %></th>
+        <th><%= sort_link @conn, @rummage, [field: :gh_created_at, name: "Created"] %></th>
+        <th><%= sort_link @conn, @rummage, [field: :gh_updated_at, name: "Updated"] %></th>
```

Comment:
Not sure if you need these `[`s, might be nicer without them

Reply:
Oh yeah course-keyword lists :man_facepalming:. That's what we get for referencing the rummage docs.